data/tekton-pipelines/okd/windows-efi-installer-configmaps.yaml

```diff
@@ -257,6 +257,9 @@ data:
                     <AcceptEula>true</AcceptEula>
                     <FullName>AdminAccount</FullName>
                     <Organization>OrgName</Organization>
+                    <ProductKey>
+                        <Key>WX4NM-KYWYW-QJJR4-XV3QB-6VM33</Key>
```

Comment:
can we make it user configurable?maybe while creating a pipeline specifying it?
i think it's different for every windows server version

Reply:
No, we don't have a mechanism to template configmaps. 
The key here is for Windows Server 2022 Datacenter as defined in https://github.com/kubevirt/tekton-tasks-operator/blob/main/data/tekton-pipelines/okd/windows-efi-installer-configmaps.yaml#L247. In case user wants different version, user has to create new configmap with adjusted xml